src/sagas/publicChannels/publicChannels.selectors.ts

```diff
@@ -130,12 +130,102 @@ export const currentChannelDisplayableMessages = createSelector(
         id: message.id,
         type: message.type,
         message: message.message,
-        createdAt: formatMessageDisplayDate(message.createdAt),
+        createdAt: message.createdAt,
         nickname: user.username,
       };
     })
 );
 
+// returns array of messages SORTED by createdAt
+export const currentChannelMessagesSortedByTime = createSelector(
```

Comment:
Duplicated selector. This is already being done in validCurrentChannelMessages selector

Reply:
Yet I wonder if it wouldn't be better if you first grouped the messages and then sorted those groups